app/api/service.py

```diff
@@ -0,0 +1,26 @@
+from flask_rest_jsonapi import ResourceDetail, ResourceList
+
+from app.api.bootstrap import api
+from app.api.schema.services import ServiceSchema
+from app.models import db
+from app.models.service import Service
+
+
+class ServiceList(ResourceList):
+    """
+    List services
+    """
+    decorators = (api.has_permission('is_admin', methods="GET"),)
+    schema = ServiceSchema
+    data_layer = {'session': db.session,
+                  'model': Service}
+
+
+class ServiceDetail(ResourceDetail):
```

Comment:
This class is very similar to the class above. Was wondering whether it could inherit from `ServiceList`, or maybe it would be an overkill?

Reply:
Please see ResourceList [GET, POST] used for (getting list of objects and create new objects) and ResourceDetail [GET, PATCH, DELETE] used for (getting, updating and deleting an object based on its ID) classes in package [flask-rest-jsonAPI](http://flask-rest-jsonapi.readthedocs.io/en/latest/)

Reply:
@schedutron Please review.

Reply:
@schedutron they inherit different parent classes, so we can't or shouldn't inherit from `ServiceList`